### PR TITLE
Setup GitHub actions

### PR DESCRIPTION
**Pull request by darrenodi**
In this pull requested,I 
- I setup linters for my Hello Microverse project and tested them out by creating and index.html and style .css file. 
- -I also included a .gitignore files to ignore some certain files in the directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+- HTML, CSS
+- .Stylelintrc
+- .Hintrc
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (Emmyn5600) [inline README.md:20]: - [ ] Optional suggestion

Let us modify the getting started section with our own description, I mean to show us how to get started with your project

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+- GitHub: [@githubhandle](https://github.com/darrenodi)
+- Twitter: [@twitterhandle](https://twitter.com/darrenodi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/darren-odi-404ba31b2/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (Emmyn5600) [inline README.md:39]: - [ ] Optional suggestion

Include the link with issue page

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @darrenodi 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the project requirements. Kudos
   
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (darrenodi) [reply README.md]: Done :)

File: README.md (hunk 2 above)
Comment (darrenodi) [reply README.md]: ;)

Comment (Emmyn5600) [reply README.md]: Good Job

File: README.md (hunk 1 above)
Comment (Emmyn5600) [reply README.md]: Well done

**Pull request merged**